preprocessing/constants.py

```diff
@@ -0,0 +1,175 @@
+# %% Name of different type of form
+formtype_names = ["household", "business", "service", "large_scale_farm", "local_aut"]
+
+MONTHS_PREFIX = "G_1b"
+
+# %% defaults usage time windows in a day
+USAGE_WD_DEFAULTS = {
+    "0-7": None,
+    "7-10": None,
+    "10-12": None,
+    "12-18": None,
+    "18-22": None,
+    "22-24": None,
+}
+
+# %% defaults months and day numaration
+
+months_defaults = {
+    "January": 1,
+    "February": 2,
+    "March": 3,
+    "April": 4,
+    "May": 5,
+    "June": 6,
+    "July": 7,
+    "August": 8,
+    "September": 9,
+    "October": 10,
+    "November": 11,
+    "December": 12,
+}
+
+working_day = {
+    "monday": 0,
+    "tuesday": 1,
+    "wednesday": 2,
+    "thursday": 3,
+    "friday": 4,
+    "saturday": 5,
+    "sunday": 6,
+}
+
+# %% Defaults constants used in preprocessing
+
+exc_rate = 0.05334  # $/ZAR exchange rate
```

Comment:
an exchange rate is not really constant in time

Reply:
This variable has been defined by Simone. He agrees that it is not correct to assume a constant exchange rate. Moreover since Simone didn't implemented the part of the code dedicated to the pre-processing of revenues information, when I took the code and upgraded it I didn't realize I should have taken into account the possibility of more than one currency. I just realized now that this variable is not used at all and that I should implement the possibility of different currencies that have to be converted. It should be put in the to-do list.